### PR TITLE
DHFPROD-8521: Re Add Run Flows From Run Tile

### PR DESCRIPTION
**Pull request by MSBrignoni**
### Description
Re Add Run Flows From Run Tile

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests  

- ##### Reviewer:

- [x] Reviewed Tests
- [x] Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Run.test.tsx [hunk 1]
```diff
@@ -148,8 +148,8 @@ describe("Verify load step failures in a flow", () => {
     // Click disclosure icon
     fireEvent.click(document.querySelector(".accordion-button"));
     let runButton = await getByLabelText("runStep-failedIngest");
-    fireEvent.mouseOver(getAllByLabelText("icon: play-circle")[0]); //temporarily fixing for DHFPROD-7820, change back to 1 eventually
-    await waitForElement(() => getByText(RunToolTips.ingestionStep));
+    fireEvent.mouseOver(getAllByLabelText("icon: play-circle")[1]); //temporarily fixing for DHFPROD-7820, change back to 1 eventually
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/pages/Run.test.tsx:151]: Remove the comment if we are reverting it to 1 now :)

File: marklogic-data-hub-central/ui/src/pages/Run.tsx [hunk 2]
```diff
@@ -92,13 +93,15 @@ const Run = (props) => {
...
   const [running, setRunning] = useState<any[]>([]);
+  const [isStepRunning, setIsStepRunning] = useState(false);
   const [uploadError, setUploadError] = useState("");
   const [openJobResponse, setOpenJobResponse] = useState<boolean>(false);
   const [jobId, setJobId] = useState<string>("");
 
   const [successModal, setSuccessModal] = useState<successModalType>({...defaultSuccessModal});
   const [errorModal, setErrorModal] = useState<errorModalType>({...defaultErrorModal});
   const [failedModal, setFailedModal] = useState<failedModalType>({...defaultFailedModal});
+  //const [successModalMultipleSteps, setSuccessModalMultipleSteps] = useState(false);
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/pages/Run.tsx:104]: Clean unused code

**Review by lucianaabc-ml — DISMISSED** [review]

File: marklogic-data-hub-central/ui/src/components/flows/flows.scss [hunk 3]
```diff
@@ -7,14 +9,20 @@
 }
 
 #stepsDropdown {
-    .ant-dropdown-trigger  {
-        padding: 0px !important;
-        width: 23%;
-        margin-right: 2.4vw;
-        &:hover{
-            background-color: #E9F7FE;
+        height: 32px;
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/src/components/flows/flows.scss:12]: Please fix indentation here.

File: marklogic-data-hub-central/ui/src/pages/Run.test.tsx (hunk 1 above)
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/pages/Run.test.tsx:151]: Thanks @aniamy !, i will take note of all the comments and solve this in the PR [7187](https://project.marklogic.com/jira/browse/DHFPROD-7187)

**Review by briantang — CHANGES_REQUESTED** [review]
@MSBrignoni looks good but there is one related e2e failure. It is expecting the spinner to appear after the flow is run. I think it's because previously Akshay used a placeholder spinner there, but you are a step ahead with putting in the modal. Let's just update the e2e to check for that modal.

One other small thing, the divider between the gear icon and the "Run Flow" does not appear until I click the gear. Is that expected, or can we have the divider there at all times?

Screen Shot 2022-03-10 at 10 19 37 AM
Screen Shot 2022-03-10 at 10 19 44 AM

**Comment by lucianaabc-ml** [comment]
> @MSBrignoni looks good but there is one related e2e failure. It is expecting the spinner to appear after the flow is run. I think it's because previously Akshay used a placeholder spinner there, but you are a step ahead with putting in the modal. Let's just update the e2e to check for that modal.

Hey, Brian! I already [fixed that e2e on my PR](https://github.com/marklogic/marklogic-data-hub/pull/6585/files#diff-f86a5b64212fc1674aea11de20f0743016bbc139c1254d6735177ebac9a270e8) since it was related to the modal.

File: marklogic-data-hub-central/ui/src/components/flows/flows.scss (hunk 3 above)
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/components/flows/flows.scss]: Done!

File: marklogic-data-hub-central/ui/src/pages/Run.tsx (hunk 2 above)
Comment (MSBrignoni) [reply marklogic-data-hub-central/ui/src/pages/Run.tsx:104]: Just in case will remove this in the next PR related with this, perhaps we will need this.

**Review by briantang — DISMISSED** [review]
Looks great now!

**Comment by MSBrignoni** [comment]
> @MSBrignoni looks good but there is one related e2e failure. It is expecting the spinner to appear after the flow is run. I think it's because previously Akshay used a placeholder spinner there, but you are a step ahead with putting in the modal. Let's just update the e2e to check for that modal.
> 
> One other small thing, the divider between the gear icon and the "Run Flow" does not appear until I click the gear. Is that expected, or can we have the divider there at all times?
> 
> Screen Shot 2022-03-10 at 10 19 37 AM
> 
> Screen Shot 2022-03-10 at 10 19 44 AM

Thank you @briantang , I fixed the button. Regarding the tests, will be skipped for now as we discussed, is a work in progress.
image

**Review by aniamy — DISMISSED** [review]

**Comment by briantang** [comment]
One test failed because a flow was not created due to the skipped test. Added the temporary fix until the test is unskipped in Luciana's PR.

**Review by briantang — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Pull request merged**